pyproject.toml

```diff
@@ -24,6 +24,8 @@ dependencies = [
   "transformers",
   "eml_parser",
   "bs4",
+  "dict2xml",
```

Comment:
which one do you want to use? I suppose `dict2xml` can be removed from the dependencies?